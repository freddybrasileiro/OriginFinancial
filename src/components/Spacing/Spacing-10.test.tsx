import * as React from 'react';
import { shallow } from 'enzyme';
import { Spacing } from './index';

describe('Spacing', () => {
  it('Should render the Spacing component with vertical 10', () => {
    expect(shallow(<Spacing vertical={10} />)).toMatchSnapshot();
  });
});
