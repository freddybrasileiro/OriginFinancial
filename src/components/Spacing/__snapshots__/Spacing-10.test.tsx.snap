// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacing Should render the Spacing component with vertical 10 1`] = `ShallowWrapper {}`;
